### DISCUSSION TITLE
Timeline on the release of the v2 version of PnP.PowerShell

### DISCUSSION THREAD

**Discussion by martinlingstuyl** [pnp/powershell#2895] [General]
Question: do you have any timeline on the v2 version of PnP.PowerShell? 
We want to update a few Azure Functions to v4, but because of the PnP dependency this is not possible. 
The only solution is to install the nightly version, which is not what you would typically want to do on production apps.

**Comment by gautamdsheth** [comment]
Very soon :) 

It also requires PnP Framework and PnP Core SDK to be released as well, but we are hoping to get it out very very soon !!

**Reply by martinlingstuyl** [reply]
Weeks? Days?

**Reply by gautamdsheth** [reply]
Leaning more towards days.
